app/App/Account/Account/Balances/Balance/index.js

```diff
@@ -39,7 +39,7 @@ class Balance extends React.Component {
     const { name: chainName = '', isTestnet = false } = chain
     const chainColor = this.store('main.networksMeta.ethereum', chainId, 'primaryColor')
 
-    const ethMatch = logoURI?.includes('/coins/images/279/large/ethereum.png')
+    const ethMatch = logoURI && logoURI.includes('/coins/images/279/large/ethereum.png')
```

Comment:
I don't think we need to change this. is it breaking on your branch?

Reply:
https://github.com/floating/frame/blob/develop/app/App/Account/Account/Balances/Balance/index.js#L42